iamy/models.go

```diff
@@ -113,6 +113,15 @@ type Role struct {
 	Policies                 []string        `json:"Policies,omitempty"`
 }
 
+type InstanceProfile struct {
+	iamService `json:"-"`
+	Roles      []string `json:"Roles,omitempty"`
+}
+
+func (ip InstanceProfile) ResourceType() string {
+	return "instance_profile"
```

Comment:
I think we'll want to follow the ARN format `instance-profile`

Reply:
c351b24